deliver/lib/deliver/commands_generator.rb

```diff
@@ -41,7 +41,7 @@ def run
       program :description, Deliver::DESCRIPTION
       program :help, 'Author', 'Felix Krause <deliver@krausefx.com>'
       program :help, 'Website', 'https://fastlane.tools'
-      program :help, 'GitHub', 'https://github.com/fastlane/fastlane/tree/master/deliver#readme'
+      program :help, 'GitHub', 'https://docs.fastlane.tools/actions/deliver/'
```

Comment:
Here too. Search for `program :help, 'GitHub'` and replace with `program :help, 'Documentation'`

Reply:
done.